src/Native/Runtime/eetype.cpp

```diff
@@ -16,6 +16,10 @@
 
 #pragma warning(disable:4127) // C4127: conditional expression is constant
 
+#ifdef CORERT
```

Comment:
Nit - this can be in main.cpp, right next to the initialization:
```
extern "C" void * g_pDispatchMapTemporaryWorkaround; 
void * g_pDispatchMapTemporaryWorkaround; 
```